commit 3abf9c9204e2fcbc1910deb102efab4ab9d8c149
Author: Niklas Haas <git@nand.wakku.to>
Date:   Wed Jun 8 17:32:40 2016 +0200

vo_opengl: don't constantly resize the output FBO

Commit 883d3114 seems to have (accidentally?) dropped the FBOTEX_FUZZY
from the output_fbo resize, which means that current master will keep
resizing and resizing the FBO as you change the window size, introducing
severe memory leaking after a while. (Not sure why that would cause
memory leaks, but I blame nvidia)

Either way, it's bad for performance too, so it's worth fixing.

diff --git a/video/out/opengl/video.c b/video/out/opengl/video.c
index bd1eb89..6c7646b 100644
--- a/video/out/opengl/video.c
+++ b/video/out/opengl/video.c
@@ -2791,7 +2791,7 @@ void gl_video_render_frame(struct gl_video *p, struct vo_frame *frame, int fbo)
                 {
                     fbotex_change(&p->output_fbo, p->gl, p->log,
                                   p->vp_w, abs(p->vp_h),
-                                  p->opts.fbo_format, 0);
+                                  p->opts.fbo_format, FBOTEX_FUZZY);
                     dest_fbo = p->output_fbo.fbo;
                     p->output_fbo_valid = true;
                 }
